src/pages/staking/index.js

```diff
@@ -1,422 +1,758 @@
-              <Translation id="page-staking-more-sharding" />
-            </Link>
-          </StyledCard>
-        </CardContainer>
-      </Content>
-    </Page>
+              </div>
+              <div style={{ gridArea: "pool-cta" }}>
+                <StyledButtonLink
+                  onClick={() => {
+                    trackCustomEvent({
+                      eventCategory: `Staking Comparison`,
+                      eventAction: `Clicked`,
+                      eventName: "clicked pooled staking",
+                    })
+                  }}
+                  to="/staking/pools/"
+                >
+                  More on pooled staking
+                </StyledButtonLink>
+              </div>
+            </ComparisonGrid>
+          </Content>
+          <Divider />
+          <StakingCommunityCallout id={tocItems.joinTheCommunity.id} />
+          <Content>
+            <h2 id={tocItems.faq.id}>{tocItems.faq.title}</h2>
+            <ExpandableCard title="What is a validator?">
+              A <em>validator</em> is a virtual entity that lives on the Beacon
+              Chain, represented by a balance, public key, and other properties.
+              A <em>validator client</em> is the software that acts on behalf of
+              the validator by holding and using its private key. A single
+              validator client can hold many key pairs, controlling many
+              validators. Each key-pair associated with a validator requires
+              32 ETH to be activated. More ETH deposited to a single set of keys
+              does not increase rewards potential, as each validator is limited
+              to an{" "}
+              <a href="https://www.attestant.io/posts/understanding-validator-effective-balance/">
+                effective balance
+              </a>{" "}
+              of 32 ETH. This means that staking is done in 32 ETH increments,
+              each with it's own set of keys and balance. Do not deposit more
+              than 32 ETH for a single validator. It will be locked until the
+              planned Shanghai update. If solo staking seems too demanding you
+              you, consider using a{" "}
+              <a href="/staking/saas/">staking-as-a-service</a> provider, or if
+              you're working with less than 32 ETH, check out the{" "}
+              <a href="/staking/pools/">staking pools</a>.
+            </ExpandableCard>
+            <ExpandableCard title="Why do I need to have funds at stake?">
+              As a validator you have the ability to propose and attest to
+              blocks for the network. To prevent dishonest behavior users must
+              have their funds at stake. This allows the protocol to penalize
+              malicious actors. Staking is a means to keep you honest, as your
+              actions will have financial consequences.
+            </ExpandableCard>
+            <ExpandableCard title="Can I buy 'Eth2'?">
```

Comment:
Is this intended to be here? Not sure it fits.

Reply:
Ah, I think I see why you added this - I'm guessing confusion around e.g. Coinbase's ETH2 staking token?

In that case, I would edit the answer here to mention that like we do on https://ethereum.org/en/security/#eth2-token-scam

Maybe something like:

```
<p>
There is no 'Eth2' token native to the protocol, as the native token ether (ETH) will not change with the transition to proof-of-stake.
</p>
<p>
There are derivative tokens/tickers that may represent staked ETH (ie. rETH from Rocket Pool, stETH from Lido, ETH2 from Coinbase). Learn more about <Link to="/staking/pools/">staking pools</Link>
</p>
```

Reply:
@samajammin Yep, can remove if it's undesirable, but thought the staking landing page would be a place where some may go looking to "buy eth2" for the upgrade. Was just trying to offer one more attempt to limit the confusion around this name, in relation to staking.

Reply:
Oops... and GitHub wasn't refreshing.. replied to that before your second message loaded. Will touch this up.